src/main/java/it/gov/pagopa/pu/migration/connector/fileshare/FileShareServiceImpl.java

```diff
@@ -16,7 +18,17 @@ public FileShareServiceImpl(FileShareClient client) {
 
   @Override
   public Long uploadIngestionFlowFile(Long organizationId, IngestionFlowFileType ingestionFlowFileType, Resource file, String accessToken) {
-    return client.uploadIngestionFlowFile(organizationId, ingestionFlowFileType, file, accessToken);
+    try {
+      return client.uploadIngestionFlowFile(organizationId, ingestionFlowFileType, file, accessToken);
+    } catch (HttpClientErrorException e) {
+      if (e.getStatusCode() == HttpStatus.BAD_REQUEST) {
+        e.getResponseBodyAsString();
+        if (e.getResponseBodyAsString().contains("\"code\":\"INVALID_FILE\"")) {
```

Comment:
This way, if we prettyPrint the response this check would no more works!
you could use getBodyAsClass using the ErrorDTO class provided with the fileshare openApi... or you should use a regexp to consider white spaces, or you should just check for the presence of the string `"INVALID_FILE"`